wrappers/leveldb/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required (VERSION 3.10)
```

Comment:
The root CMakeLists.txt defines the required version to be 3.5.
Can you check if 3.10 is required? If so I should update the version in the root folder.